### DISCUSSION TITLE
all stacks in STACK2 folder are .nc type?

### DISCUSSION THREAD

**Discussion by wandrly** [ROBelgium/MSNoise#324] [Q&A] [answered]
Hi there,

As the title states, after stacking, all of the stacks in STACK2/ have an extension of .nc - to me this says incomplete, but perhaps I'm missing something? The 001/ folder in STACKS has separate folders for each station pair, and then the daily .SAC files, which is the expected behaviour.

Any hints on what went wrong or how to fix it?

Thanks,
wandrly

**Comment by ThomasLecocq** [comment, marked as answer]
The development version you are using is changing the default (previous) behaviour, and the NC (netCDF) files are the new default format for the successive steps.
